worker-tasks/sbt

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env bash
+#
+# A more capable sbt runner, coincidentally also called sbt.
+# Author: Paul Phillips <paulp@improving.org>
+
```

Comment:
I installed `sbt` using the official APT repository on Bintray within the Spark container image in https://github.com/azavea/raster-foundry/commit/eb00dac947c75dc291bac5a6c8408ead3baf6d9a.